src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.csproj

```diff
@@ -4,7 +4,6 @@
     <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
     <EnableDefaultItems>false</EnableDefaultItems>
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">2.0.0-beta-001776-00</RuntimeFrameworkVersion>
```

Comment:
While it probably isn't necessary we should probably start passing this version in to avoid restoring it.